src/Access/Quota.h

```diff
@@ -76,6 +76,7 @@ struct Quota : public IAccessEntity
         NONE,       /// All users share the same quota.
         USER_NAME,  /// Connections with the same user name share the same quota.
         IP_ADDRESS, /// Connections from the same IP share the same quota.
+        FORWARDED_IP_ADDRESS, /// Use X-Forwarded-For HTTP header instead of IP address.
```

Comment:
The name `FORWARDED_IP_ADDRESS` doesn't seem to be clear without comments. Maybe it's better to rename it into something like `PROXY_ORIGIN_IP_ADDRESS`?

Reply:
The address that proxy server forwards to us... also the analogy with X-Forwarded-For HTTP header.

Reply:
I understood, it just seems that the name `FORWARDED_IP_ADDRESS` is not very good because it requires to know that HTTP header and doesn't even contain the word `PROXY`.